src/main/java/uk/gov/ons/fwmt/job_service_v2/converter/impl/LMSConverter.java

```diff
@@ -0,0 +1,119 @@
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import java.time.ZoneId;
+import java.util.GregorianCalendar;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.ons.fwmt.job_service_v2.utils.TMJobConverter.addAdditionalProperty;
+import static uk.gov.ons.fwmt.job_service_v2.utils.TMJobConverter.addAddressLines;
+import static uk.gov.ons.fwmt.job_service_v2.utils.TMJobConverter.checkNumberOfAddressLines;
+
+@Component("LMS")
+public class LMSConverter implements TMConverter {
+
+  private static final String DESCRIPTION = "OHS";
+  private static final int DURATION = 0;
+  private static final boolean STATUS = false;
+  private static final boolean EMERGENCY = false;
+  private static final boolean DISPATCHED = false;
+  private static final boolean APPOINTMENT_PENDING = false;
+  private static final String SKILL = "OHS";
+  private static final String WORK_TYPE = "OHS";
+  private static final String DEFAULT_WORLD = "Default";
+  private static final String MOD_WORLD = "MOD WORLD";
+
+  public CreateJobRequest convert(FWMTCreateJobRequest ingest) {
+    CreateJobRequest createJobRequest =  new CreateJobRequest();
+    JobType job = new JobType();
+
+    job.setIdentity(new JobIdentityType());
+    job.setSkills(new SkillCollectionType());
+    job.setLocation(new LocationType());
+    job.setContact(new ContactInfoType());
+    job.setWorld(new WorldIdentityType());
+
+    job.getIdentity().setReference(ingest.getJobIdentity());
+    job.getLocation().setAddressDetail(new AddressDetailType());
+    job.getLocation().getAddressDetail().setLines(new AddressDetailType.Lines());
+    List<String> addressLines = job.getLocation().getAddressDetail().getLines().getAddressLine();
+
+    addAddressLines(addressLines, ingest.getAddress().getLine1());
+    addAddressLines(addressLines, ingest.getAddress().getLine2());
+    addAddressLines(addressLines, ingest.getAddress().getLine3());
+    addAddressLines(addressLines, ingest.getAddress().getLine4());
+    addAddressLines(addressLines, ingest.getAddress().getTownName());
+    checkNumberOfAddressLines(addressLines);
+
+    //TODO: Map Co-ordinates
+    //job.getLocation().getAddressDetail().setGeoX(new JAXBElement<Float>().setValue(ingest.getAddress().getLongitude().floatValue());
+
+    job.getLocation().getAddressDetail().setPostCode(ingest.getAddress().getPostCode());
+
+    if(StringUtils.isNotBlank(ingest.getSurveyType())) {
+      job.setWorkType(ingest.getSurveyType());
+    } else {
+      job.setWorkType(WORK_TYPE);
+    }
+
+    GregorianCalendar dueDateCalendar = GregorianCalendar
+        .from(ingest.getDueDate().atTime(23, 59, 59).atZone(ZoneId.of("UTC")));
+    try {
+      job.setDueDate(DatatypeFactory.newInstance().newXMLGregorianCalendar(dueDateCalendar));
+    } catch (DatatypeConfigurationException e) {
+      e.printStackTrace();
+      //TODO: Handle exception properly
```

Comment:
Throw CTPException passing in the DatatypeConfigurationException exception as its 2nd parameter